Tests/Shared.Specs/StringAssertionSpecs.cs

```diff
@@ -1867,6 +1869,816 @@ public void When_string_containment_is_asserted_and_actual_value_is_null_then_it
 #endif
         }
 
+        #endregion
+		
+		#region Default with times
+
+        [InlineData("ABCDEF", "BCD", 1)]
+        [InlineData("ABCDEBCDF", "BCD", 2)]
+        [InlineData("ABXYZCDXYZEBCDFXYZ", "XYZ", 3)]
```

Comment:
🤔 Hmm, aren't you duplicating the number of scenarios? We do gray box testing. So you don't have to academically prove that it works. Just be smart about the scenarios. Same with the other tests.

Reply:
Ok, I have heard about gray box testing, but I'm not sure what you exactly mean (in this case).
So please confirm.

Should I remove tests from `Default with times` region?
These tests are exactly the same as in `Exactly` region (but for different API).

Should I have less input data?

Reply:
Just the `InlineData`.

Reply:
Ok, thanks